ddtrace/tracer/abandoned_spans_test.go

```diff
@@ -240,3 +240,7 @@ func TestDebugAbandonedSpansOff(t *testing.T) {
 		s.Finish()
 	})
 }
+
```

Comment:
oops might want to delete this commented test?

Reply:
oops indeed, thanks!